components/docs-chef-io/content/habitat/application_lifecycle_hooks.md

```diff
@@ -142,3 +142,13 @@ File location: `<plan>/hooks/post-run`. The post run hook will get executed afte
 ### post-stop
 
 File location: `<plan>/hooks/post-stop`. The post-stop hook will get executed after service has been stopped successfully. You may use this hook to undo what the `init` hook has done.
+
+n `install` hook, unlike other hooks with the exception of the `uninstall` hook, will not have access to any census data exposed via binds or the `svc` namespace. Also, configuration in `svc_config_path` is not accessible to an `install` hook. If an `install` hook needs to use templated configuration files, templates located in the `svc_config_install_path` may be referenced. This location will contain rendered templates in a package's `config_install` folder. Finally, any configuration updates made during a service's runtime that would alter an `install` hook or any configuration template in `svc_config_install_path` will not cause a service to reload.
```

Comment:
```suggestion
An `install` hook, unlike other hooks with the exception of the `uninstall` hook, will not have access to any census data exposed via binds or the `svc` namespace. Also, configuration in `svc_config_path` is not accessible to an `install` hook. If an `install` hook needs to use templated configuration files, templates located in the `svc_config_install_path` may be referenced. This location will contain rendered templates in a package's `config_install` folder. Finally, any configuration updates made during a service's runtime that would alter an `install` hook or any configuration template in `svc_config_install_path` will not cause a service to reload.
```

Reply:
Was this paragraph intended to be duplicated from further up in the doc. It seems out of place here.